scripts/dataset-download.sh

```diff
@@ -19,6 +20,7 @@ function download_dataset
   full_dataset_name=$1
   restore_path=$2
   dataset_name=${full_dataset_name%-*}
+  dataset_neo4j_version=$3
```

Comment:
Swap both lines, parameters then calculation based on parameters for readibility